cfg/builder/builder_walk.cc

```diff
@@ -58,7 +58,35 @@ LocalRef global2Local(CFGContext cctx, core::SymbolRef what) {
     return alias;
 }
 
-LocalRef unresolvedIdent2Local(CFGContext cctx, const ast::UnresolvedIdent &id) {
+core::ClassOrModuleRef resolveBlockContext(CFGContext cctx, const core::ClassOrModuleRef outterClassOrModule) {
+    auto methodRef = outterClassOrModule.data(cctx.ctx)->findMethodTransitive(cctx.ctx, cctx.link->fun);
```

Comment:
This change seems not great from a performance perspective.  I don't know what better options look like, but `findMethodTransitive` use is a red flag.